### PR TITLE
Fix some non-standard comments

### PR DESCRIPTION
**Pull request by laglangyue**
## description
Fix some non-standard comments
use tool of idea 'replaceAll', and check  them in local
## why need
it will cause some method can't generate doc
please view the site `https://pekko.apache.org/japi/pekko/current/org/apache/pekko/stream/scaladsl/Source.html`
image

and some changes maybe not included in doc, I also believe it is necessary to be more standardized for comment
## test
just add `*` for comment, not need test local,it will passed by CI

### CODE REVIEW

File: actor/src/main/scala/org/apache/pekko/actor/FSM.scala [hunk 1]
```diff
@@ -743,29 +743,29 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
     case x    => x.stateData
   }
 
-  /*
+  /**
```
Comment (pjfanning) [inline actor/src/main/scala/org/apache/pekko/actor/FSM.scala:746]: please remove this change - this comment is not a method or field comment

File: actor/src/main/scala/org/apache/pekko/actor/FSM.scala [hunk 2]
```diff
@@ -743,29 +743,29 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
...
    * ****************************************************************
    *                PRIVATE IMPLEMENTATION DETAILS
    * ****************************************************************
    */
 
   private[pekko] def debugEvent: Boolean = false
 
-  /*
+  /**
```
Comment (pjfanning) [inline actor/src/main/scala/org/apache/pekko/actor/FSM.scala:754]: actually, I don't see the point in changing the field comments

Can you just change function comments and only function comments?

**Review by pjfanning — CHANGES_REQUESTED** [review]

File: actor/src/main/scala/org/apache/pekko/actor/FSM.scala (hunk 2 above)
Comment (laglangyue) [reply actor/src/main/scala/org/apache/pekko/actor/FSM.scala:754]: ok, I will check it again

**Review by pjfanning — APPROVED** [review]
lgtm

**Comment by He-Pin** [comment]
Thanks , for new contributor from China:)

**Pull request merged**